gui/qt/main_window.py

```diff
@@ -4307,7 +4307,7 @@ def cpfp(self, parent_tx, new_tx):
         vbox.addWidget(WWLabel(_(msg2)))
         grid = QGridLayout()
         grid.addWidget(QLabel(_('Total size') + ':'), 0, 0)
-        grid.addWidget(QLabel('%d bytes'% total_size), 0, 1)
+        grid.addWidget(QLabel(_(f'{total_size} bytes')), 0, 1)
```

Comment:
hehe i keep catching small stuff like this as i read code for cashaccts.. totally understand your nit here. :)

Reply:
I need to be able to translate "bytes" to "Bytes" for German :)

Reply:
hehe :)